Src/java/tools/cql-formatter/src/test/java/org/cqframework/cql/tools/formatter/CqlFormatterVisitorTest.java

```diff
@@ -62,6 +63,8 @@ public void TestFormatterSpecific() throws IOException {
         Assert.assertFalse(inError);
         runTest("git-issue-349.cql");
         Assert.assertFalse(inError);
+        runTest("cqf-838.cql");
```

Comment:
CQF-838 is not an issue in this github repository. Is there a github issue this is linked to?